frontend/src/config/keybindings/index.ts

```diff
@@ -0,0 +1,94 @@
+import keybindings from './keybindings.json';
+import { createShortcutIncludingModifierKeysFromKeyEvent } from '@/utils/key-combinations';
+import React from 'react';
+
+export const KEYBINDING_SCOPE_GLOBAL = 'global';
+export const KEYBINDING_SCOPE_PLANTS_LAYER = 'plants_layer';
+
+type KeyBinding = Record<string, string[]>;
+
+type KeyBindingConfig = {
```

Comment:
consistently plural?

```suggestion
type KeyBindingsConfig = {
```